Makefile

```diff
@@ -26,7 +26,7 @@ ci: ci-black ci-flake8 ci-test ## Check black, flake8, and run unit tests
 	@echo "CI sucessful"
 
 isort: build-docker  ## Runs isort to sorts imports
-	docker run -v ${PWD}:/mnt -w /mnt -it $(IMAGE) isort -rc $(PROJ_DIR)
+	docker run -v ${PWD}:/mnt -w /mnt -it $(IMAGE) isort $(PROJ_DIR)
```

Comment:
This is unrelated. New version of `isort` doesn't require this flag. This removes an `isort` warning.